<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sine current source control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="usbConnectBtn">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="csControlGB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Current source control</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0" alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="calibrationBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Calibration</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0" alignment="Qt::AlignHCenter">
            <widget class="QCheckBox" name="powerCtrlCB">
             <property name="text">
              <string>Power on</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="amplitudeGB">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>Amplitude</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0" colspan="3">
                 <widget class="QRadioButton" name="peakRB">
                  <property name="text">
                   <string>Peak</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3" colspan="2">
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <pointsize>20</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>A</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3" colspan="2">
                 <widget class="QRadioButton" name="rmsRB">
                  <property name="text">
                   <string>RMS</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1" colspan="2" alignment="Qt::AlignRight">
                 <widget class="QDoubleSpinBox" name="amplitudeSpinBox">
                  <property name="font">
                   <font>
                    <pointsize>20</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="decimals">
                   <number>1</number>
                  </property>
                  <property name="minimum">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>5.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.500000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string> </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
